packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1501,15 +1501,18 @@ export default class Meeting extends StatelessWebexPlugin {
       : this.destination;
     const destinationType = isStartingSpaceInstantV2Meeting ? _MEETING_LINK_ : this.destinationType;
 
-    const timeLeft = this.getPermissionTokenTimeLeftInSec();
+    const {permissionTokenTimeLeft, permissionExpiryTimestamp, currentTime} =
+      this.getPermissionTokenTimeLeftInSec();
 
     LoggerProxy.logger.info(
-      `Meeting:index#refreshPermissionToken --> refreshing permission token, destinationType=${destinationType}, timeLeft=${timeLeft}, reason=${reason}`
+      `Meeting:index#refreshPermissionToken --> refreshing permission token, destinationType=${destinationType}, timeLeft=${permissionTokenTimeLeft}, permissionTokenExpiry=${permissionExpiryTimestamp}, currentTimestamp=${currentTime},reason=${reason}`
     );
 
     Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.PERMISSION_TOKEN_REFRESH, {
       correlationId: this.correlationId,
-      timeLeft,
+      permissionTokenTimeLeft,
```

Comment:
it would be better to keep the existing name of the property `timeLeft` so that we have continuity in the metrics graphs